packages/pigeon/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## 14.0.1
+## 15.0.0
 
+* **Breaking Change** [kotlin] Updates Flutter API to use new errorClassName.
```

Comment:
This will need to be a new entry, not overwrite the previous.
```
## 15.0.0

* **Breaking Change** [kotlin] Updates Flutter API to use new errorClassName.

## 14.0.1

* Updates minimum supported SDK version to Flutter 3.10/Dart 3.0.
* Updates issue_tracker link.
```

Reply:
Whoops good catch, thank you!